lib/pivotal-tracker/proxy.rb

```diff
@@ -0,0 +1,17 @@
+class PivotalTracker::Proxy
```

Comment:
Its initialize process does not take a hash of attributes, so I don't believe that it should inherit from resource